go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/matcornic/hermes
+
+require (
+	github.com/Masterminds/semver v1.4.2 // indirect
+	github.com/Masterminds/sprig v2.16.0+incompatible
+	github.com/aokoli/goutils v1.0.1 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/google/uuid v1.0.0 // indirect
+	github.com/huandu/xstrings v1.2.0 // indirect
+	github.com/imdario/mergo v0.3.6
+	github.com/jaytaylor/html2text v0.0.0-20180606194806-57d518f124b0
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/mattn/go-runewidth v0.0.3 // indirect
+	github.com/olekukonko/tablewriter v0.0.0-20180912035003-be2c049b30cc // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/russross/blackfriday/v2 v2.0.1
```

Comment:
This should change to not include v2 probably.

Reply:
why? this is the expected behaviour. the current version of hermes is also including v2 of blackfriday.